client/src/components/Members/Community/WebMembersList.js

```diff
@@ -1,29 +1,30 @@
-export const WebMembersLists = [{
-            id: 0,
-            img: './images/Members/Web and App Dev/pankaj.jfif',
-            name: 'Pankaj Waghela',
-            position: 'Community Head',
-            mail: 'vpankaj1998official@gmail.com',
-            github: '',
-            linkedin: 'linkedin.com/in/iampankajvaghela'
-      },
+export const WebMembersLists = [
+      // {
+      //       id: 0,
+      //       img: './images/Members/WebandAppDev/pankaj.jfif',
+      //       name: 'Pankaj Waghela',

```

Comment:
Pankaj Waghela is also not holding any position. Might remove this as well.

Reply:
Ohh about that, yeah the comments were redundant but i thought maybe not get rid of the photos cuz maybe we could create like a wall of ex-members in the future. Let me see this anyway.

Reply:
Oh yea, remove the photos as well if you will. It'd marginally reduce the deployment size.